src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Diagnostics;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace Dynamo.DocumentationBrowser
+{
+    /// <summary>
+    /// Interaction logic for DocumentationBrowserView.xaml
+    /// </summary>
+    public partial class DocumentationBrowserView : UserControl, IDisposable
+    {
+        private const string LOCAL_FILES_URI_IN_BROWSER = "about:blank";
+        private readonly DocumentationBrowserViewModel viewModel;
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="p">ViewLoadedParams</param>
+        public DocumentationBrowserView(DocumentationBrowserViewModel viewModel)
+        {
+            InitializeComponent();
+            this.DataContext = viewModel;
+            this.viewModel = viewModel;
+
+            // subscribe to the link changed event on the view model
+            // so we know when to navigate to a new documentation page/document
+            viewModel.LinkChanged += NavigateToPage;
+
+            // handle browser component events & disable certain features that are not needed
+            this.documentationBrowser.AllowDrop = false;
+            this.documentationBrowser.NavigationStarting += NavigationStarting;
+            this.documentationBrowser.NavigationCompleted += NavigationCompleted;
```

Comment:
Does the dispose function of `documentationBrowser` take care of ubsubscribing?

Reply:
It probably doesn't, so added unsubscribing to the dispose of the view
b8646480ec7148f85ed7e4c21eec1f8094d6b41e